src/main/drivers/compass.h

```diff
@@ -20,10 +20,11 @@
 #include "sensor.h"
 
 typedef struct magDev_s {
-    sensorInitFuncPtr init;                                 // initialize function
-    sensorReadFuncPtr read;                                 // read 3 axis data function
+    sensorMagInitFuncPtr init;  // initialize function
+    sensorMagReadFuncPtr read;  // read 3 axis data function
```

Comment:
Other typedefs for sensor init/read (acc and gyro) use `FuncPtr` suffix - I suggest for the scope of this PR we keep things consistent.